shed/tests/test_streams.py

```diff
@@ -1015,3 +1015,39 @@ def test_event_contents_fail(exp_db, start_uid1):
     s = exp_db.restream(ih1, fill=True)
     for a in s:
         source.emit(a)
+
+
+def test_ingest_streams():
+    """ Ensure that stream curation works as intended"""
+    s = es.EventStream()
+    # try both dict and None type
+    doc1 = ('start', None)
+    doc2 = ('start', {})
+
+    doc3 = (('start', {}), ('start', {}))
+
+    doc4 = (('start', {}), ('start', ({}, {})))
```

Comment:
the same. you modified the line to 
```python
if isinstance(docs, tuple) and len(docs) == 1:
    docs = docs[0]
```
(there was no if statement there last time I read it I believe)
so I think it should be fine. I would still try that case just in case. It is slightly different logic from the other docs being tested. (but won't increase the coverage)